packages/vsx-registry/src/browser/vsx-extensions-contribution.ts

```diff
@@ -222,4 +243,28 @@ export class VSXExtensionsContribution extends AbstractViewContribution<VSXExten
     protected copyExtensionId(extension: VSXExtension): void {
         this.clipboardService.writeText(extension.id);
     }
+
+    protected async showRecommendedToast(): Promise<void> {
+        if (!this.preferenceService.get(IGNORE_RECOMMENDATIONS_ID, false)) {
+            const recommended = new Set([...this.model.recommended]);
+            for (const installed of this.model.installed) {
+                recommended.delete(installed);
+            }
+            if (recommended.size) {
+                const userResponse = await this.messageService.info('Would you like to install recommended plugins?', 'Install', 'Show Recommended');
```

Comment:
I think I did this because there's some hard-coded max width on the buttons in a toast and 'Recommendations' wouldn't fit.
![image](https://user-images.githubusercontent.com/62660806/121263107-c749cd00-c87a-11eb-8896-bfc2fc459dab.png)

Happy to fix that value, though, if you'd like.

Reply:
That's unfortunate :) We can leave it as is then :+1: